packages/wallet-connect-sdk-wallet-core/src/sdk.ts

```diff
@@ -142,22 +144,24 @@ export class WcWalletSDK {
         throw new Error('Initialization timeout has been reached')
       })
 
-      const client = await SignClient.init({
-        signConfig: {
-          disableRequestQueue: true,
-        },
-        ...this.clientOptions,
+      const core = new Core({
+        projectId: this.clientOptions.projectId,
+      })
+
+      const web3wallet = await Web3Wallet.init({
+        ...(this.clientOptions as unknown as Web3WalletTypes.Options),
```

Comment:
Is there a good reason for not changing `clientOptions` from `WalletConnectTypes.SignClientTypes.Options` to `Web3WalletTypes.Options`?

Reply:
The ``WalletConnectTypes.SignClientTypes.Options`` extends ``CoreTypes.Options`` and has the projectId within it, while the ``Web3WalletTypes.Options`` does not extend. In ``Web3WalletTypes.Options``, the projectId is within the ``core`` variable. This would affect the wallet initialization [options](https://github.com/CityOfZion/wallet-connect-sdk/blob/bfd888b76060dc19cab96eb96a2e3127b966a9ab/examples/wc-wallet-react/src/index.tsx#L19C7-L19C16) declaration.